ui/src/screen/network.ts

```diff
@@ -0,0 +1,18 @@
+import { derived } from "svelte/store";
+
+import * as localPeer from "../localPeer";
+import * as remote from "../remote";
+
+interface Screen {
+  status: localPeer.Status;
+}
+
+export const store = derived(
+  [localPeer.status],
+  ([status], set: (screen: remote.Data<Screen>) => void): void => {
+    if (status.status === remote.Status.Success) {
+      set({ status: remote.Status.Success, data: { status: status.data } });
+    }
+  },
+  { status: remote.Status.Loading }
+);
```

Comment:
Could you explain why we need this derived store? It looks to me like we can use `localPeer.status` in the network screen directly since it contains the exact same information. This would allow us to eliminate this file.